__tests__/lib/openapi-to-json-schema.test.js

```diff
@@ -738,6 +738,7 @@ describe('`example` / `examples` support', () => {
 
       const schema = operation.getParametersAsJsonSchema()[0].schema;
 
+      expect(schema.components).toBeUndefined();
```

Comment:
Components is/should be undefined here because the OAS it's testing has been dereferenced and there aren't any circular `$ref`s in it.